app/src/main/java/com/rage/clamber/Networking/Requests/NewCommentRequest.java

```diff
@@ -0,0 +1,43 @@
+package com.rage.clamber.Networking.Requests;
+
+/**
+ * Comment body for server posts
+ */
+public class NewCommentRequest {
+    protected String username;
+    protected int climbId;
+    protected String commmentText;
+    protected long dateText;
```

Comment:
In addition to the rename, go ahead an add a comment explaining how the date is represented